webapp/src/components/App/Tasks/JobTableView.tsx

```diff
@@ -84,16 +90,20 @@ function JobTableView(props: Props) {
         [
           filterRunningStatus && (({ status }: TaskView) => status === "running"),
           filterType && (({ type }: TaskView) => type === filterType),
+          filterUser && (({ userName }: TaskView) => isSearchMatching(filterUser, userName)),
```

Comment:
Since `userName` is optional in `TaskView`, this could pass `undefined` to `isSearchMatching`. Should add a null check to be safe I guess
```ts
filterUser && (({ userName }: TaskView) => userName && isSearchMatching(filterUser, userName)),

```
or simply pass a default value to the predicate `isSearchMatching(filterUser, userName || "")`